test/functional/page_objects/dashboard_page.ts

```diff
@@ -215,6 +215,8 @@ export function DashboardPageProvider({ getService, getPageObjects }: FtrProvide
 
     public async clickNewDashboard() {
       await listingTable.clickNewButton('createDashboardPromptButton');
+      // make sure the dashboard page is shown
+      await this.waitForRenderComplete();
```

Comment:
When switching to a dashboard, we introduce an additional delay with this change because another network roundtrip is necessary. This stabilizes the test so it doesn't fail.